packages/jsii-rosetta/lib/languages/python.ts

```diff
@@ -833,6 +839,10 @@ const TOKEN_REWRITES: { [key: string]: string } = {
   false: 'False',
 };
 
+const IDENTIFIER_KEYWORDS: { [key: string]: string } = {
+  lambda: 'awslambda',
```

Comment:
This is too CDK-specific for JSII. Why not `lambda_` ? It's not great but a simple convention.

In fact, any keyword can be non-keyworded by appending `_`.